pyproject.toml

```diff
@@ -13,6 +13,31 @@ ignore = [
     "PLR0913", # Too many arguments to function call
     "PLC1901", # `<VAR> != ''` can be simplified to `<VAR>` as an empty string is falsey
     "PLR2004", # Magic value used in comparison
+    "F541",
```

Comment:
Can you leave a comment here that clarifies that we intend to eliminate these added rules as opposed to the existing ones which we indefinitely turned off

Reply:
added!